test/unit/domain/ledger/imbalance-calculator.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { toSats } from "@domain/bitcoin"
+import { ImbalanceCalculator } from "@domain/ledger/imbalance-calculator"
+import { WalletCurrency } from "@domain/shared"
+
+let btcWallet: Wallet
+
+beforeAll(() => {
+  btcWallet = {
```

Comment:
this doesn't mutate iso there is no need for a `beforeAll` - can just assign it as const.
Actually I cannot even see you using anything other than the `wallet.id` so perhaps you only need that.